data/config-schema/src/main/resources/net/corda/schema/configuration/rpc/1.0/corda.rpc.json

```
@@ -48,6 +48,20 @@
           "default": 12000
         }
       }
+    },
+    "websocket": {
+      "description": "Configuration settings for WebSockets.",
+      "type": "object",
+      "default": {},
+      "properties": {
+        "idleTimeoutMs": {
+          "description": "Amount of time in milliseconds for a websocket connection to be idle before timing out and closing the connection.",
```

Value of `0` will be interpreted as "never". From `org.eclipse.jetty.io.EndPoint#setIdleTimeout`: `Params:
idleTimeout – the idle timeout in MS. Timeout <= 0 implies an infinite timeout`.

Therefore:
```suggestion
          "description": "Amount of time in milliseconds for a websocket connection to be idle (i.e. no messages are either sent or received) before timing out and closing the connection. Value of 0 means such idle check will be disabled.",
```

---

Well spotted, thanks!